### DISCUSSION TITLE
ComfyUI --force-fp16

### DISCUSSION THREAD

**Discussion by x-GeneSin-x** [Stability-AI/StableSwarmUI#336] [Q&A] [answered]
Hey! Can you help me, how to start stableswarmUI if I want ComfyUI to start up with --force-fp16? The only way it works by added the flag to comfyUI's batch, but that way it runs as standalone.

**Comment by mcmonkey4eva** [comment, marked as answer]
go to Server -> Backends -> click the edit icon on the comfy backend -> put that in `ExtraArgs` -> hit save

**Reply by x-GeneSin-x** [reply]
thank you
